include/rocksdb/db.h

```diff
@@ -379,7 +380,10 @@ class DB {
   // options.sync=true.
   // Returns OK on success, non-OK on failure.
   // Note: consider setting options.sync = true.
-  virtual Status Write(const WriteOptions& options, WriteBatch* updates) = 0;
+  virtual Status Write(const WriteOptions& options, WriteBatch* updates, uint64_t* seq) = 0;
+  virtual Status Write(const WriteOptions& options, WriteBatch* updates)  {
```

Comment:
@tabokie It is an override of the old interface

Reply:
Oh, right.